engines/dfc_provider/app/controllers/dfc_provider/api/persons_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# Controller used to provide the Persons API for the DFC application
+module DfcProvider
+  module Api
+    class PersonsController < DfcProvider::Api::BaseController
+      skip_before_action :check_enterprise
+
+      before_action :check_user_accessibility
+
+      def show
+        render json: user, serializer: DfcProvider::PersonSerializer
+      end
+
+      private
+
+      def user
+        @user ||= Spree::User.find(params[:id])
```

Comment:
So this controller is only supposed to allow access to show the current user? I'm not sure we need to fetch the user record again here, it's already have it available via `#current_user`, right?

Reply:
It was refactored few times, and I kept it this way in case an user could access the catalogue of an other user in the future, I wanted to keep the routes as open as possible to avoid later refactoring, always annoying to refactor routes.